tests/unit/capi/GEOSOffsetCurveTest.cpp

```diff
@@ -104,7 +104,7 @@ void object::test<3>()
 {
      checkOffset(
         "LINESTRING(0 0, 10 0, 10 10)",
-        "LINESTRING (12 10, 12 0, 10 -2, 0 -2)",
+        "LINESTRING (0 -2, 10 -2, 10.390180644 -1.9615705608, 10.7653668647 -1.847759065, 11.111140466 -1.6629392246, 11.4142135624 -1.4142135624, 11.6629392246 -1.111140466, 11.847759065 -0.7653668647, 11.9615705608 -0.390180644, 12 0, 12 10)",
         -2, 1, GEOSBUF_JOIN_ROUND, 2,
         0.000001
```

Comment:
Note that the last argument is a tolerance, so there's no need for more than 6 decimal digits with the current tolerance.